src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
```

Comment:
Well `loginPrompt`is the user configuration, so wouldn't want to rename that - for a start should be in the JSDoc block (not sure why ESlint didn't complain that it's not). Not sure how else to make it clearer?